### ISSUE TITLE
How to add MIC loss in DACS training

### ISSUE THREAD

**Issue by yuheyuan** [lhoyer/MIC#53]
![image](https://github.com/lhoyer/MIC/assets/39433014/2410f4c5-4293-4e63-ae9e-8f0f85625805)
I see you do experience , add MIC in DACS. 
I want to know how to do it. Thank you very much.
Is it using 
```python
               # Use 1x source image and 1x target image for MIC
        elif self.mask_mode in ['separate', 'separateaug']:
            assert img.shape[0] == 2  # 确定source的batch为2
            masked_img = torch.stack([img[0], target_img[0]])  
            masked_lbl = torch.stack(
                [gt_semantic_seg[0], masked_plabel[0].unsqueeze(0)])  
            gt_pixel_weight = torch.ones(masked_pweight[0].shape, device=dev)  
                [gt_pixel_weight, masked_pweight[0]])  
```
and apply  'aug' in the img
```python
# Apply color augmentation
        if 'aug' in self.mask_mode:
            strong_parameters = {
                'mix': None,  # change here?
                'color_jitter': random.uniform(0, 1),
                'color_jitter_s': self.color_jitter_s,
                'color_jitter_p': self.color_jitter_p,
                'blur': random.uniform(0, 1),
                'mean': means[0].unsqueeze(0),
                'std': stds[0].unsqueeze(0)
            }
            masked_img, _ = strong_transform(
                strong_parameters, data=masked_img.clone())
```

**Comment by lhoyer** [comment]
You can find the code to generate the configs for all the different UDA methods with MIC (including DACS) in https://github.com/lhoyer/MIC/blob/81782e070f5e9f1911c9eeaa0353129b4f073850/seg/experiments.py#L417

The readme https://github.com/lhoyer/MIC/tree/master/seg#training provides the relevant instructions on how to train with experiment.py.
